src/ui/App.tsx

```diff
@@ -23,6 +25,23 @@ const App = () => {
   const toastMsg = useAppSelector(getToastMsg);
   const [showScan, setShowScan] = useState(false);
   const [showToast, setShowToast] = useState(false);
+  const browserPreview = new URLSearchParams(window.location.search).has(
+    "mobile"
```

Comment:
imo `browserPreview` is a better query param. And we don't check for `=true` we just check it exists, but guess it doesn't matter.

`mobile` sounds like something that happens when deployed on mobile

Reply:
^ @jimcase

Reply:
I think he suggested that because it would sound more like 'mobile view'.